chart/values.schema.json

```diff
@@ -552,6 +552,11 @@
                                 "IfNotPresent"
                             ],
                             "default": "IfNotPresent"
+                        },
+                        "extraEnv": {
+                            "description": "Add additional env vars to flower.",
+                            "type": "array",
+                            "default": []
```

Comment:
You should add "items" here so we can properly validate what the user supplies here. e.g:
https://github.com/apache/airflow/blob/2af19f16a4d94e749bbf6c7c4704e02aac35fc11/chart/values.schema.json#L695-L710

Reply:
Have added "items" as standard validation for envVar
```
"items": {
                                "$ref": "#/definitions/io.k8s.api.core.v1.EnvVar"
                            }
```